src/messages/MOSDPing.h

```diff
@@ -61,12 +61,29 @@ class MOSDPing : public Message {
   uuid_d fsid;
   epoch_t map_epoch = 0;
   __u8 op = 0;
-  utime_t stamp;
+  utime_t ping_stamp;               ///< when the PING was sent
+  ceph::signedspan mono_ping_stamp; ///< relative to sender's clock
```

Comment:
I started by trying to use a mono_clock::time_point, but that doesn't have an encode/decode.

This series deliberately uses mono times relative to the osd startup time so that in all cases (testing, production) the clock base is (very) different, and we can be sure the delta arithmetic etc works correctly.  I'm not sure that works as well with time_point.